include/seqan3/std/charconv

```diff
@@ -859,6 +859,23 @@ inline std::to_chars_result to_chars(char * first, char * last, value_type value
     return seqan3::detail::to_chars_itoa(first, last, value, is_signed<value_type>());
 }
 
+//!\brief std::to_chars overload for floating point via a stringstream.
+//!\ingroup charconv
+template <seqan3::FloatingPoint floating_point_type>
+inline std::to_chars_result to_chars(char * first, char * last, floating_point_type value, int = 10) noexcept
```

Comment:
`value` can be const
and what is the `int = 10` for?

Reply:
well, it is a copy anyway but I guess const does not hurt.

The int = 10 is for compatibility of the function signature. to_chars is documented to have a parameter `base`, but since I wrote a dirty, inefficient version of this function I do not use this parameter. To avoid unused parameter warnings it does not have a name, and in order to be callable without the base parameter I default the non-existent parameter... didn't know this works until I tried it ^^